helloworld-jest-test-sample/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+	"compilerOptions": {
+		"module": "commonjs",
+		"target": "es6",
```

Comment:
This can be es2019 now to match up with https://github.com/microsoft/vscode-extension-samples/blob/master/webpack-sample/tsconfig.json

Reply:
Once changed bump VSCode up to “^1.49” in package.json